test/agents.ts

```diff
@@ -53,6 +53,18 @@ describe('agents', () => {
         'HTTPS_PROXY',
       ];
 
+      const noProxyEnvVars = ['no_proxy', 'NO_PROXY'];
+
+      function cleanProxyEnv() {
```

Comment:
I think you can use `sandbox.stub.value`, and avoid needing to write this logic yourself, given that we have sinon as a dep already:

https://github.com/googleapis/repo-automation-bots/blob/6c8571c77dcaa93436b462d9be351ca5b735b435/packages/owl-bot/test/owl-bot.ts#L43

Reply:
Nice, thanks for the tip - updated